account_bank_statement_import_online/models/online_bank_statement_provider.py

```diff
@@ -246,6 +261,13 @@ def _pull(self, date_since, date_until):
                                 )
                             )
                         continue
+
+                    if isinstance(date, datetime):
+                        if date.tzinfo is None:
+                            date = date.replace(tzinfo=utc)
+                        date = date.astimezone(provider_tz).replace(tzinfo=None)
+                        line_values['date'] = date
```

Comment:
`line_values['date'] = self._get_statement_date(date, None)`

Reply:
Confusing optimisation, and defies flexibility of `_get_statement_date` override